ConsoleApp1/Program.cs

```diff
@@ -1,15 +1,118 @@
+            }
+
+            else
+            {
+                var fexpression = (int)((x + y) * (Math.Pow(z, 2) + 1));
+                Console.WriteLine("First expression: " + fexpression);
+
+                var sexpression = ((x % z) - 1) * (Math.Sqrt(y));
+                Console.WriteLine("Second expression: " + (Math.Round(sexpression, 2)));
+
+                var texpression = ((x + y) * (y + z)) / (Math.Pow(z, 3));
+                Console.WriteLine("Third expression: " + (Math.Round(texpression, 2)));
+                Console.ReadKey();
+            }
+
+            //Exercise 2
+            Console.WriteLine("Exercise 2. Press any key");
+            Console.ReadKey();
+            Console.WriteLine("Enter length");
+            int length = Convert.ToInt32(Console.ReadLine());
+            Console.WriteLine("Enter width");
+            int width = Convert.ToInt32(Console.ReadLine());
+            Console.WriteLine("Enter height");
+            int height = Convert.ToInt32(Console.ReadLine());
+
+            if(length < 0 ^ width < 0 ^ height < 0)
+            {
+                Console.WriteLine("Incorrect data");
+            }
+
+            else
+            {
+                var diagonal = Math.Sqrt(Math.Pow(length, 2) + Math.Pow(width, 2) + Math.Pow(height, 2));
+                Console.WriteLine("Parallelepiped diagonal length = " + (Math.Round(diagonal, 2)));
+                Console.ReadKey();
+            }
+
+            //Exercise 3
+            Console.WriteLine("Exercise 3. Press any key");
+            Console.ReadKey();
+            Console.WriteLine("Enter string 1");
+            var str1 = Console.ReadLine().ToUpper();
+            Console.WriteLine("Enter string 2");
+            var str2 = Console.ReadLine().ToUpper();
+            Console.WriteLine("Enter string 3");
+            var str3 = Console.ReadLine().ToUpper();
+
+            if (str1.StartsWith(str2))
+            {
+                Console.WriteLine("First string starts from second string: " + str2);
+            }
+            else
+            {
+                Console.WriteLine("First string doesn't start from second string.");
+            }
+
+            if (str1.EndsWith(str3))
+            {
+                Console.WriteLine("First string ends with third string: " + str3);
+            }
+            else
+            {
+                Console.WriteLine("First string doesn't end with third string.");
+            }
+            Console.ReadKey();
+
+            //Exercise 4
+            Console.WriteLine("Exercise 4. Press any key");
+            Console.ReadKey();
+            Console.WriteLine("Enter string 1");
+            String string1 = Console.ReadLine();
+            Console.WriteLine("Enter string 2");
+            String string2 = Console.ReadLine();
+            Console.WriteLine("Enter symbol");
+            char ch = Convert.ToChar(Console.ReadLine());
```

Comment:
What if will input more than one char?

Maybe there is a method in Console class that will help us to read the char?

Reply:
I tried to use the console.readkey, but every time the program stopped at this point.